Python/bytecodes.c

```diff
@@ -1149,16 +1169,35 @@ dummy_func(
             }
         }
 
-        inst(LOAD_NAME, ( -- v)) {
-            PyObject *name = GETITEM(frame->f_code->co_names, oparg);
-            PyObject *locals = LOCALS();
-            if (locals == NULL) {
+        op(_LOAD_NAME_INTRO, (-- class_dict, name)) {
```

Comment:
I think `class_dict` might be a confusing name here, because `LOAD_NAME` is also used in module scope, where this will be a module dict, not a class dict. But of course in the case of `LOAD_CLASS_OR_GLOBAL` it will always be a class dict. I would still name it something more generic so it isn't inaccurate for the `LOAD_NAME` case: maybe `mod_or_class_dict`?

Reply:
Renamed to `mod_or_class_dict`.